lisa/tests/scheduler/eas_behaviour.py

```diff
@@ -73,6 +89,11 @@ def get_big_duty_cycle(cls, plat_info):
         second to biggest CPUs in the system, thereby forcing up-migration
         while minimizing the thermal impact.
         """
+        # big_task_duty_cycle is set when the platform requires a specific
+        # value for the big task duty cycle.
+        if big_task_duty_cycle:
```

Comment:
test for `is None` and add an `else` clause with the rest of the function in it.